dist/index.js

```diff
@@ -40,6 +40,11 @@ module.exports = class Deviance {
             });
         }
 
+        const { threshold } = this.settings.regression;
+        if (typeof threshold !== 'number' || threshold < 0 || threshold > 1) {
```

Comment:
Should this be using the `hasValidThreshold` helper?

Reply:
yes, corrected